Hunting Queries/OracleDatabaseAudit/OracleDBAuditUsersNewPrivilegesAdded.yaml

```diff
@@ -0,0 +1,31 @@
+id: 6be9634e-6661-4bff-98f1-4a743d7231a5
+name: OracleDBAudit - Users with new privileges
+description: |
+  'Query for searching user accounts whith new privileges.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - InitialAccess
+  - PrivilegeEscalation
+query: |
+  let lbtime_60d = 60d;

```

Comment:
30d max recommended as a default for hunting queries, but not required. Just remember this could cause performance issues.

Reply:
updated

Reply:
looks like this did not get updated, if you want to leave at 60d, just respond with that. :)

Reply:
fixed